anonymous_tokens/cpp/crypto/rsa_blind_signer.h

```diff
@@ -25,13 +25,14 @@
 #include "anonymous_tokens/proto/anonymous_tokens.pb.h"
 
 
+
 namespace anonymous_tokens {
 
 // The RSA SSA (Signature Schemes with Appendix) using PSS (Probabilistic
 // Signature Scheme) encoding is defined at
 // https://tools.ietf.org/html/rfc8017#section-8.1). This implementation uses
 // Boring SSL for the underlying cryptographic operations.
-class RsaBlindSigner : public BlindSigner {
+class  RsaBlindSigner : public BlindSigner {
```

Comment:
Is this adding an extra space?

Reply:
Consistent with other files due to copybara script issue. Will resolve later.